app/javascript/react/components/MovieShowContainer.js

```diff
@@ -22,19 +25,73 @@ const MovieShowContainer = (props) => {
       })
       .catch((error) => console.error(`Error in fetch: ${error.message}`))
   }, [])
+
+  if (movie.title && !reviews){
+    setReviews(movie.reviews)
+  }
```

Comment:
This conditional should be within the useEffect, after you `setMovie`, you can conditionally check if the movie has any reviews, if so `setReviews` otherwise, do nothing